src/sentry/api/serializers/models/group.py

```diff
@@ -638,42 +644,74 @@ def serialize(self, obj, attrs, user):
 
 
 class GroupSerializerSnuba(GroupSerializerBase):
-    def __init__(self, environment_ids=None, start=None, end=None, snuba_filters=None):
+    skip_snuba_fields = {
+        "query",
+        "status",
+        "bookmarked_by",
+        "assigned_to",
+        "unassigned",
+        "subscribed_by",
+        "active_at",
+        "first_release",
+        "first_seen",
+        "message",
```

Comment:
Probably worth removing `message` based on our discussion this morning

Reply:
Removed :-)